CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 * Allow to listen to the symfony console events
 * Add a `compile` command that puts together a customizable PHP binary with a
   repacked castor app into one executable file
+* Distribute `castor.linux.amd64` automatically with the release
```

Comment:
Can we also add linux.arm64 and darwin.arm64?

Reply:
nope :( Github does not support yet ARM64.
I asked to join [the beta](https://resources.github.com/devops/accelerate-your-cicd-with-arm-and-gpu-runners-in-github-actions/), but for now it's not possible

Reply:
You can build for ARM using QEMU. It's just a bit slower, but I think it only runs on tag/release so might be fine?

Reply:
I don't know how to do that :/ Do you want to give it a try? (but after this PR is merged, we are still not sure we want to merge it)

Reply:
Yes, let's have a look after this has been merged.